plugins/mongo_db/mongo_db_state.cpp

```diff
@@ -239,11 +239,11 @@ namespace mongo_db {
 
             format_value(body, "last_post", account.last_post);
 
-#ifndef IS_LOW_MEM
-            auto& account_metadata = db_.get<account_metadata_object, by_account>(account.name);
-            
-            format_value(body, "json_metadata", account_metadata.json_metadata);
-#endif
+            if (db_.store_account_metadata()) {
+                auto& account_metadata = db_.get<account_metadata_object, by_account>(account.name);
```

Comment:
In this place will be segfault

Reply:
Why? How to reproduce?

Reply:
Flag can be enabled between restarts of the daemon.
And some accounts can has metadata, other can be without metadata.

And additional case:
- Start daemon with flag
- Add account, it will store metadata
- Restart daemon without flag
- Modify account, it should remove metadata

Reply:
@afalaleev Already understood. You could remove comment, then I'll remove my own.